### PR TITLE
Tips.xml: Old string changed

### PR DESCRIPTION
**Pull request by PushKK**
1. Old menu changed:
Database Processing > Family Tree Processing
Compare and Merge... > Merge...
2. Add string "The function ".

### CODE REVIEW

**Comment by Nick-Hall** [comment]
Merge actually works for all object types now.

File: data/tips.xml [hunk 1]
```diff
@@ -39,9 +39,9 @@
 
 <tip number="20"><b>Unsure of a Date?</b><br/>If you're unsure about the date an event occurred, Gramps allows you to enter a wide range of date formats based on a guess or an estimate. For instance, &quot;about 1908&quot; is a valid entry for a birth date in Gramps. Click the Date button next to the date field and see the Gramps Manual to learn more.</tip>
 
-<tip number="21"><b>Duplicate Entries</b><br/>&quot;Tools &gt; Database Processing &gt; Find Possible Duplicate People...&quot; allows you to locate (and merge) entries of the same person entered more than once in the database.</tip>
+<tip number="21"><b>Duplicate Entries</b><br/>The function &quot;Tools &gt; Family Tree Processing &gt; Find Possible Duplicate People...&quot; allows you to locate (and merge) entries of the same person entered more than once in the database.</tip>
```
Comment (Nick-Hall) [inline data/tips.xml:42]: Perhaps change "The function" to "The tool" here.

File: data/tips.xml [hunk 2]
```diff
@@ -39,9 +39,9 @@
...
 
-<tip number="22"><b>Merging Entries</b><br/>The function &quot;Edit &gt; Compare and Merge...&quot; allows you to combine separately listed people into one. Select the second entry by holding the Control key as you click. This is very useful for combining two databases with overlapping people, or combining erroneously entered differing names for one individual. This also works for the Places, Sources and Repositories views.</tip>
+<tip number="22"><b>Merging Entries</b><br/>The function &quot;Edit &gt; Merge...&quot; allows you to combine separately listed people into one. Select the second entry by holding the Control key as you click. This is very useful for combining two databases with overlapping people, or combining erroneously entered differing names for one individual. This also works for the Places, Sources and Repositories views.</tip>
```
Comment (Nick-Hall) [inline data/tips.xml:44]: Change "listed people" to "listed objects".  Change the last sentence to "This works in all list views and with all object types.".

**Review by Nick-Hall — CHANGES_REQUESTED** [review]
Just a few changes as previously mentioned.

File: data/tips.xml (hunk 2 above)
Comment (PushKK) [reply data/tips.xml:44]: I check menu points when translate. In Gramps 'Edit &gt; Compare and Merge...' not present, then I change this to real text.

**Comment by Nick-Hall** [comment]
Merged with PR #1226 in commit 1d1d3707ce4f5c27ae8fe93aca6aa46115be11ce.

**Comment by Nick-Hall** [comment]
Merged with PR #1226 in commit 1d1d3707ce4f5c27ae8fe93aca6aa46115be11ce.

**Comment by emyoulation** [comment]
A new proposed   tips.xml   is attached in a Note to:
https://gramps-project.org/bugs/view.php?id=10546

The original request was for changing URLS in this file from    http   to   https

**Pull request closed without merging**